src/frontend/src/test-e2e/selenium.test.ts

```diff
@@ -41,6 +41,15 @@ const DEMO_APP_URL = "http://localhost:8080/";
 const DEVICE_NAME1 = "Virtual WebAuthn device";
 const DEVICE_NAME2 = "Other WebAuthn device";
 
+const screenshots = new Screenshots("screenshots");
+
+async function screenshot(
```

Comment:
do we really need this function? we could just use 'screenshots.take' directly

Reply:
I tried keeping the diff size low :)